gobcore/model/gobmodel.json

```diff
@@ -1390,6 +1398,24 @@
             "type": "GOB.String",
             "description": "Amsterdamse identificerende code."
           },
+          "cbs_nummer": {
+            "type": "GOB.String",
+            "description": "CBS-nummer."
+          },
+          "woningvoorraad": {
+            "type": "GOB.String",
+            "description": "Geeft aan of een ver­blijfs­ob­ject bij de woningvoorraad hoort."
+          },
+          "financieringscode": {
+            "type": "GOB.JSON",
+            "description": "Geeft aan op welke wijze een woning gefinancierd is bij de bouw.",
+            "code": {
```

Comment:
shouldn't this be within "attributes": {} ?

Reply:
Sharp, thanks. Copied in the wrong place, added it.